pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -255,6 +201,8 @@ spec:
           - name: pathInRepo
             value: tasks/push-snapshot/push-snapshot.yaml
       params:
+        - name: tagPrefix
+          value: ""
```

Comment:
@theflockers could you please help with the value of `tagPrefix`?

Reply:
for the record, me and happy discussed the following in slack:

- tagPrefix should be also added as pipeline parameter and the value should be $(params.tagPrefix)
- Another pipeline parameter should be added - timestampFormat, so the timestamp can be changed from the default of unix timestamp set in the task.

Reply:
With the API changes, we won't be passing pipeline parameters anymore. Everything comes from the CRs. So these values should live in the `spec.data` section on one of them. Maybe the RPA?

Reply:
It will, I'd guess in a followup (it doesn't have to be part of this). For example, here is how we get the pyxis info now (this is the RPA)
```
spec:
  data:
    pyxis:
      server: stage
      secret: mysecret
```
So we will need to make more additions to support tags and things like `addGitShaTag` which probably means reworking `push-snapshot` to pull them from `data.json` but we haven't done it yet

Reply:
That what I was thinking. Should we for now keep it as is (with params) and once the API is merged we rework both (as both pipelines  - the old push and this new one - will require changes)?

Reply:
I'm not opposed to that idea